### PR TITLE
task 19 completed

### PR DESCRIPTION
**Pull request by kajihan**
database-aws
main_output

### CODE REVIEW

File: src/main/java/pro/java/hw19/Main.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    private static void getLessonByID(LessonDao lessonDao, int lessonID) {
+        Lesson retrievedLesson = lessonDao.getLessonById(lessonID);
+        String message = lessonDao.getLessonById(lessonID) != null
+                ? "Lesson by id " + lessonID + ": " + retrievedLesson.getName() + ", " + "Homework: " + retrievedLesson.getHomework().getName()
+                : "Lesson by id " + lessonID + " not found";
+        System.out.println(message);
+    }
+
+    private static void getAllLessons(LessonDao lessonDao) {
```
Comment (kartius) [inline src/main/java/pro/java/hw19/Main.java:65]: Better, don't use a void for a method type. You need to return the result

File: src/main/java/pro/java/hw19/Main.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        List<Lesson> allLessons = lessonDao.getAllLessons();
+        allLessons.forEach(System.out::println);
+    }
+
+    private static void getAllHomeworks(HomeworkDao homeworkDao) {
```
Comment (kartius) [inline src/main/java/pro/java/hw19/Main.java:70]: And the same for all methods

File: src/main/java/pro/java/hw19/dao/LessonDao.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LessonDao {
+    public void addLesson(Lesson lesson) {
```
Comment (kartius) [inline src/main/java/pro/java/hw19/dao/LessonDao.java]: Better to get connector via constructor only ones instead of check it in each method

Comment (kajihan) [reply src/main/java/pro/java/hw19/dao/LessonDao.java]: fixed

**Pull request merged**